rcl/test/rcl/test_init.cpp

```diff
@@ -271,3 +271,17 @@ TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_rcl_get_instance_id)
   ret = rcl_context_fini(&context);
   EXPECT_EQ(ret, RCL_RET_OK);
 }
+
+TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_rcl_init_options_access) {
```

Comment:
While you're editing this file, you can add two lines of coverage by adding a test elsewhere that runs test_init with a non-zero argc and a nullptr as the argv.

I think ultimately, we'll have to mock the functions `rcl_init` calls to get 100% of coverage though.

https://ci.ros2.org/job/ci_linux_coverage/115/cobertura/src_ros2_rcl_rcl_src_rcl/init_c/

Reply:
Was this added?

Reply:
I added them before your check, but github is not working properly. I closed and reopened the PR and now they appear.